### PR TITLE
feat: Add github action that will add reviewer

### PR DESCRIPTION
**Pull request by opokornyy**
**What this PR does / why we need it**:

This PR should automate adding given users to monitoring PRs.

**Release note**:
```
None
```

### CODE REVIEW

**Comment by akrejcir** [comment]
Have you tested this in your own fork?

**Comment by opokornyy** [comment]
> Have you tested this in your own fork?

Yes, I did.

File: .github/workflows/add-reviewer-monitoring.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Add Reviewer
+        id: add_reviewer
+        run: gh pr edit $PULL_REQUEST_NUMBER --add-reviewer sradco
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+          PULL_REQUEST_NUMBER: ${{  github.event.pull_request.number }}
```
Comment (0xFelix) [inline .github/workflows/add-reviewer-monitoring.yaml:26]: Can you inline this?

File: .github/workflows/add-reviewer-monitoring.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      - docs/metrics.md
+      - tests/metrics_test_utils.go
+      - tools/metricsdocs/*
+
+jobs:
+  add_reviewer:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
```
Comment (0xFelix) [inline .github/workflows/add-reviewer-monitoring.yaml:18]: Do we really need to checkout the code first?

File: .github/workflows/add-reviewer-monitoring.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+name: Add reviewer on monitoring PRs
```
Comment (0xFelix) [inline .github/workflows/add-reviewer-monitoring.yaml:1]: Can you name it `Add reviewer from sig-monitoring to PR`?

File: .github/workflows/add-reviewer-monitoring.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+on:
+  pull_request:
+    types: [opened]
```
Comment (0xFelix) [inline .github/workflows/add-reviewer-monitoring.yaml:5]: Is there a type for when an existing PR is updated to change a file from this list?

File: .github/workflows/add-reviewer-monitoring.yaml (hunk 2 above)
Comment (opokornyy) [reply .github/workflows/add-reviewer-monitoring.yaml:18]: Yes, without checkout the `gh` command does not work properly.

File: .github/workflows/add-reviewer-monitoring.yaml (hunk 4 above)
Comment (opokornyy) [reply .github/workflows/add-reviewer-monitoring.yaml]: Yes, I have missed this case, not it should work also on updating existing PR

**Comment by lyarwood** [comment]
/approve

It's a hackaround, still would prefer to see this done with OWNERS but as we saw it's awkward and we can't enforce reviewers so this will have to do for now.

**Pull request merged**